main.py

```diff
@@ -0,0 +1,110 @@
+from session import mySession
+import toml
+with open('C:\learn_req\data.toml') as file:
```

Comment:
You're not using this code anywhere. Delete it maybe? Also delete the commented code if it's not needed anymore.